models/configuration.go

```diff
@@ -40,17 +17,11 @@ type Configuration struct {
 	// The password for the PCF API.
 	PCFPassword string `json:"pcf_password"`
 
-	// verifyOpts is intentionally lower-cased so it won't be stored in JSON.
-	// Instead, this struct is expected to be created from NewConfiguration
-	// so that it'll populate this field.
-	verifyOpts *x509.VerifyOptions
-}
+	// The maximum minutes old a login request's signing time can be.
+	// This is configurable because in some test environments we found as much as 2 hours of clock drift.
+	LoginMaxMinOld int `json:"login_max_minutes_old"`
```

Comment:
I added this field because in testing, I found that their popular `cf dev` environment freezes the time to match the time its virtual box was spun up. I didn't want to inhibit customer testing, so added the ability to configure how old the signature could be.